### DISCUSSION TITLE
What about have a ideia?

### DISCUSSION THREAD

**Discussion by marcosalvesdev** [community/community#65036] [Other Feature Feedback, Questions, & Ideas]
Recently I have been asking me how to get a idea about some project and make it be real. For some reason it have being hard to me found something new to be really interesting to build. I have some skills and don't know what to do with it because it's like I didn't find something that I really enjoy to do. What do you think guys I have to do now? I just want have fun building something and not just build it because it's my job function. What is the road to discovery something that will really make my heart beat fast again? Have you ever had to face these questions? If yes, what did you do? Hope my question helps other friends here <3

**Comment by Zerotask** [comment]
For me, when I was a teenager, I played a space browser game. I found an open source project which was very similar to that game. So for me it would be a good idea to make something similar, with my current knowledge and current technologies.
Think back, how you started your path. Is there something you really liked?

Here are also some resources:
- https://github.com/florinpop17/app-ideas
- https://github.com/The-Cool-Coders/Project-Ideas-And-Resources

**Comment by LeandroLuna** [comment]
I completely understand where you're coming from. Finding that spark of inspiration for a new project can indeed be challenging. What's great is that you're not alone in facing this dilemma, and it's wonderful that you're seeking advice to reignite your passion for creating something meaningful.

In my experience, the road to discovering a project that truly excites you often involves a combination of exploration, self-reflection, and embracing your interests. Here are a few steps you might consider taking:

- Explore Diverse Topics: Don't limit yourself to your current skillset. Delve into various fields, technologies, or areas that intrigue you. Sometimes, a new skill or interest can lead you to a project idea you hadn't considered before.

- Identify Personal Challenges: Reflect on any challenges or problems you face in your daily life. Often, the best projects stem from addressing real-world issues you're personally invested in.

- Community Involvement: Engage with online communities, forums, and social media platforms related to your interests. Discussions with fellow enthusiasts can often spark ideas and provide fresh perspectives.

- Passion-Driven Learning: Pursue learning opportunities that genuinely excite you. As you acquire new skills or deepen existing ones, project ideas might naturally emerge.

- Personal Projects: Start small by working on personal projects that are purely for fun or exploration. Without the pressure of being a job function, you can experiment, fail, and learn without constraints.

- Contribute to Open Source: Contributing to open-source projects allows you to work on something impactful, learn from others, and collaborate within a larger community.

- Hackathons and Challenges: Participate in hackathons or challenges related to your interests. These time-bound events can provide the motivation you need to start and complete a project.

- Seek Inspiration: Explore the work of others, whether it's through blogs, YouTube channels, or online showcases. Sometimes, seeing what others have achieved can ignite your own creativity.

It's absolutely okay to experience periods of uncertainty and creative lulls. Many people, including myself, have faced similar questions. For me, delving into new areas of interest, collaborating with others, and exploring open-source projects have been incredibly enriching experiences.

I hope this has helped you!
